Packages/com.llealloo.audiolink/Runtime/Scripts/AudioLinkController.cs

```diff
@@ -95,10 +92,12 @@ void Start()
                 }
                 else
                 {
-                    themeColorController.SetProgramVariable("audioLink", audioLink);
+                    themeColorController.audioLink = audioLink;
                     themeColorController.SendCustomEvent("UpdateAudioLinkThemeColors");
                 }
 
+                GetSettings();
```

Comment:
The ordering here intends for the reset button to reset to the settings the user has chosen in the inspector, I assume?

Reply:
Correct